frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -183,73 +294,35 @@ const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
     trimmedContent = '';
   };
   const onToggleFilename = () => setFilenameOpen(!isFilenameOpen);
-  const options = (items) =>
-    items.map((value) => {
-      return (
-        <SelectOption
-          key={value}
-          value={value}
-          className={classnames({ 'co-node-logs__log-select-option': value.length > 50 })}
-        />
-      );
-    });
-  const logLabel = t('public~Select a log file');
   const errorExists = error.length > 0;
   const logFilenamesExist = logFilenames.length > 0;
 
+  const logControls = (
```

Comment:
Use `React.useCallback` so that the component doesn't unmount/remount every render.